vnext/Scripts/StripAdditionalPlatformsFromNuspec.ps1

```diff
@@ -0,0 +1,28 @@
+# For PR builds we dont want to have to build _ALL_ the platforms just to get a nuget and test one platform.
+# this script filters the files required by the nuget, so we can produce a test nuget on a single platform
+
```

Comment:
We have a script that does something like this already (but I think it's broken). Could we reconcile the two?